scripts/train.py

```diff
@@ -0,0 +1,52 @@
+import os
+import click
+import sys
+from sklearn.externals import joblib
+from sklearn.neural_network import MLPClassifier
+from pkspace.utils.loaders import PKSpaceLoader, PKLotLoader
+
+
+@click.command()
+@click.option('--loader', '-l', type=click.Choice(['PKLot', 'PKSpace']),
+              default='PKSpace', help='Loader used to load dataset')
+@click.option('--model_type', '-mt', type=click.Choice(['MLP']), default='MLP',
+              help='Type of model to be trained')
+# @click.option('--hidden_later', '-hl', default=(15, 10),
```

Comment:
@rstevanak I think this should probably be `hidden_layers` rather than `hidden_later`. Also I would suggest to refrain from using two letters as abbreviation of longer argument form. I don't think that there is anything wrong with it but it is just not very pleasant to type when running it from command line. So in case of `--model_type` I would suggest just simple `-m`. For `--hidden_layers` I think just `-h` would suffice and for `--model_path` I would use probably `-p`.